src/pandas_profiling/profile_report.py

```diff
@@ -424,3 +426,9 @@ def _repr_html_(self) -> None:
     def __repr__(self) -> str:
         """Override so that Jupyter Notebook does not print the object."""
         return ""
+
+    def get_default_settings(self, df) -> Settings:
+        if isinstance(df, pd.DataFrame) or isinstance(df, pd.Series):
```

Comment:
isinstance's second argument can be a tuple:
`if isinstance(df, (pd.DataFrame, pd.Series))`